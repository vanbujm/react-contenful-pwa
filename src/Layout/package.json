{
  "main": "./Layout.js",
  "name": "Layout"
}