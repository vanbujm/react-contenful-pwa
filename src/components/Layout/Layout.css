.layout {
    display: flex;
    flex-direction: column;
    width: 50%;
    margin: 0 auto;
    flex: 1;
    padding: 2rem 1rem;
}