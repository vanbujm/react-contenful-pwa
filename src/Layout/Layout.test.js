import React from 'react';
import ReactDOM from 'react-dom';
import Layout from './Layout';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Layout><div>Test</div></Layout>, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('Renders children', () => {
  const tree = renderer
    .create(<Layout><div>Test</div></Layout>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
