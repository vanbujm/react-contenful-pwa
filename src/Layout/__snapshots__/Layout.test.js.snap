// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders children 1`] = `
<div
  className="layout"
>
  <div>
    Test
  </div>
</div>
`;
